next/src/components/layouts/Sections.tsx

```diff
@@ -123,33 +129,22 @@ const SectionContent = ({ section }: { section: SectionsFragment }) => {
     case 'ComponentSectionsDocuments':
       return <DocumentsSection section={section} />
 
+    case 'ComponentSectionsNumericalList':
+      return <NumericalListSection section={section} />
+
     default:
       return null
   }
 }
 
-const Section = ({ section }: { section: SectionsFragment | null }) => {
-  if (!section) return null
-
-  if (section.__typename === 'ComponentSectionsNumericalList') {
-    return <NumericalListSection section={section} />
-  }
-
-  return (
-    <SectionContainer className="pt-10 md:pt-18">
-      <SectionContent section={section} />
-    </SectionContainer>
-  )
-}
-
-const Sections = ({ sections }: { sections: (SectionsFragment | null)[] }) => {
+const Sections = ({ sections, className }: SectionsProps) => {
   return (
-    <>
+    <div className={className}>
```

Comment:
Curious if this additional div is really needed. Shouldn't we apply `className` on `SectionContent `instead? (and change `*:py-5 *:md:py-9` to `py-5 md:py-9`)